### PR TITLE
PL-288 - minor updates to support deploying in EB.

### PR DESCRIPTION
**Pull request by nergdron**
@j-martin: not sure what else I need to do to get this artifact built and pushed out to staging. can you show me tomorrow?

### CODE REVIEW

**Comment by philwhln** [comment]
Looks good to me. Will let @j-martin sign-off on this.

File: run.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+rm -f tmp/pids/server.pid
+export SECRET_KEY_BASE="$(bundle exec rake secret)"
+
+# configure database
+bundle exec rake db:create
+bundle exec rake db:migrate
+
+# run service
+bundle exec rails s -p 80 -b '0.0.0.0'
```
Comment (j-martin) [inline run.sh]: you may want to run `exec bundle ...` at the end.

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,7 @@
+def ctx = [
+  name: 'pop',
+  build: [type: 'ruby'],
+  skipIntegrationTest: true,
+]
+
+publishService([name: 'pop'])
```
Comment (j-martin) [inline Jenkinsfile]: `publishService(ctx)`

Comment (j-martin) [reply Jenkinsfile]: since you need to pass the context. You also need a `.bench.yml` to track dependencies.

File: run.sh (hunk 1 above)
Comment (nergdron) [reply run.sh]: not sure what you mean?

**Pull request merged**